### PR TITLE
Allow passing raw samples as input.

### PR DESCRIPTION
**Pull request by harrynull**
Fixes #19 .

### CODE REVIEW

**Comment by saurabhshri** [comment]
Hello @harrynull, excellent work! :) Looks perfect to me. Have you tested it by running it with a raw file?

**Comment by harrynull** [comment]
Not yet, because CCAligner didn't work in Windows at the time I did it, and I also didn't have raw file sample to test it. But I will test it later if you didn't.

**Comment by saurabhshri** [comment]
@harrynull I am sending you a raw file in Slack DM. Please test it, I don't have a windows machine to try this.

File: src/lib_ccaligner/params.cpp [hunk 1]
```diff
@@ -11,7 +11,7 @@ const std::string currentTime()
     time_t now = time(0);
     struct tm tstruct = * localtime(&now);
     std::string localTime;
-    localTime.resize(19);   //len(%d-%m-%Y-%H-%M-%S)
+    localTime.resize(32);   //len(%d-%m-%Y-%H-%M-%S)
```
Comment (saurabhshri) [inline src/lib_ccaligner/params.cpp]: Any specific reason, why this was changed? The string length is exact 19 characters?

It leads to 

```
logPath             : tempFiles/02-12-2017-11-49-42.log
phonemeLogPath      : tempFiles/phoneme-02-12-2017-11-49-42.log
```

getting changed to : 

```
logPath             : tempFiles/02-12-2017-11-52-03.log
phonemeLogPath      : tempFiles/phoneme-02-12-2017-11-52-03.log
```

Comment (harrynull) [reply src/lib_ccaligner/params.cpp]: @saurabhshri I don't know why it works in Linux, but you need at least 19+1(\0)==20 bytes to store the result.

Comment (saurabhshri) [reply src/lib_ccaligner/params.cpp]: @harrynull C++ strings do not need to end in NUL character :) 

Unlike character array, which is simply an array of characters terminated by a NUL character, a string is a class which defines objects that be represented as stream of characters. 

See the "String Length and Accessing Individual Elements" section at https://www.cprogramming.com/tutorial/string.html

**Pull request merged**